R/process_sc_all_care_home.R

```diff
@@ -19,62 +19,72 @@
 #' @family process extracts
 #'
 #' @export
-#'
+
+
 process_sc_all_care_home <- function(
     data,
     sc_demog_lookup = read_file(get_sc_demog_lookup_path()) %>% slfhelper::get_chi(),
     it_chi_deaths_data = read_file(get_slf_chi_deaths_path()),
```

Comment:
```suggestion
    it_chi_deaths_data = read_file(get_slf_chi_deaths_path()) %>% slfhelper::get_chi(),
```
need to convert to chi as this causes an error matching later on